esnext/index.html

```diff
@@ -2156,9 +2156,9 @@
 <td class="yes" data-browser="chrome81">Yes</td>
 <td class="yes unstable" data-browser="chrome83">Yes</td>
 <td class="yes unstable" data-browser="chrome84">Yes</td>
-<td class="no obsolete" data-browser="edge15">No</td>
-<td class="no obsolete" data-browser="edge17">No</td>
-<td class="no" data-browser="edge18">No</td>
+<td class="yes obsolete" data-browser="edge15">Yes</td>
+<td class="yes obsolete" data-browser="edge17">Yes</td>
+<td class="yes" data-browser="edge18">Yes</td>
```

Comment:
For example, this is supported by IE7 and we have any other data that says that newer version of Chakra don't support it.

Reply:
Isn't this issue #1118?

Reply:
Hm, possibly? If so, i guess we can close it :-)